Library/Homebrew/test/livecheck/strategy/sparkle_spec.rb

```diff
@@ -189,20 +349,20 @@
         sparkle.versions_from_content(xml[:appcast]) do |item|
           item.nice_version&.sub("1", "0")
         end,
-      ).to eq([subbed_items[0]])
+      ).to eq([subbed_versions_in_sort_order[0]])
 
       # Returning an array of strings from block
       expect(
         sparkle.versions_from_content(xml[:appcast]) do |items|
           items.map { |item| item.nice_version&.sub("1", "0") }
         end,
-      ).to eq(subbed_items)
+      ).to eq(subbed_versions_in_sort_order)
```

Comment:
```suggestion
     ).to eq(subbed_items)
```

Reverting in relation to the changes above.